src/test/kotlin/org/gradle/bot/utils/GithubKeyCheckerTest.kt

```diff
@@ -0,0 +1,23 @@
+        // test for normal
+        assertTrue(GithubKeyChecker.verifySignature(payload, "sha1=f7ceac8e913abc39bd7e318955c67db290e64f8b", "hello"));
+        assertTrue(GithubKeyChecker.verifySignature(payload, "sha1=a4847957ad54fd147c8a13c9da234b4727165018", "gradle-bot"));
+        // test for fake key
+        assertFalse(GithubKeyChecker.verifySignature(payload, "sha1=iamfakesha1aaaaaaaaaaaaaaaaaaaaaaaaaaaaa", "fackKey"));
```

Comment:
Don't use kotlin test, use JUnit 5, which has a lot of new features and advantages, for example, these `assertTrue`/`assertFalse`s can be replaced with  [ParameterizedTest](https://junit.org/junit5/docs/current/user-guide/#writing-tests-parameterized-tests).

You can find a sample here: https://github.com/gradle/gradle/blob/a808fec7031ad1e6fe483a78f61589f016560fa2/buildSrc/subprojects/plugins/src/test/kotlin/org/gradle/plugins/buildtypes/BuildTypesPluginTest.kt#L57-L68

Reply:
```suggestion
        assertFalse(GithubKeyChecker.verifySignature(payload, "sha1=iamfakesha1aaaaaaaaaaaaaaaaaaaaaaaaaaaaa", "fakeKey"));
```

I guess this is a typo...